### ISSUE TITLE
Requires react-router-dom

### ISSUE THREAD

**Issue by dugajean** [Daydreamer-riri/vite-react-ssg#34]
I'm trying to use this library for a single page without routes, but it's forcing me to install react-router-dom.

```
node:internal/modules/esm/resolve:854
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'react-router-dom' imported from /node_modules/vite-react-ssg/dist/node/cli.mjs
```

**Comment by Daydreamer-riri** [comment]
Thank you for reporting this, this is a known issue that has been fixed in the latest development branch. The next release will release this fix.

You can install it for the time being, which will not affect your client product.

**Comment by Daydreamer-riri** [comment]
Thank you for reporting this, this is a known issue that has been fixed in the latest development branch. The next release will release this fix.

You can install it for the time being, which will not affect your client product.

**Comment by Daydreamer-riri** [comment]
It will be released this month together with tanstack/router support.

**Comment by Daydreamer-riri** [comment]
Hi @dugajean,  this issue has been fixed in version 0.8.0.
